ext/parquet/go.mod

```diff
@@ -10,3 +10,5 @@ require (
 )
 
 replace github.com/bsm/feedx => ../../
+
+replace github.com/bsm/feedx/internal/pbio => ../../internal/pbio
```

Comment:
I don't understand why that would even need pbio

Reply:
It was a dep from `feedx/go.mod` (ended up in `ext/parquet/go.sum`).

Not relevant anymore.